### ISSUE TITLE
Mod Silently doesn't work

### ISSUE THREAD

**Issue by IanSimcock** [Serilum/.issue-tracker#636]
Hi Rick.

I came across your impressive list of mods yesterday and decided to try a bunch if them out. I tried the Paper Books mod first but it didn't work. There was no crash or anything, it just didn't work.

I tried throwing a number of your mods in and none of them would work. It was at that point that I noticed that my launcher, MultiMC, was unable to extract the mod name or version from your mods, and that in Minecraft the number of mods it said that it saw did not change when I added or removed your mods. I even tried a clean unmodded copy, but it didn't help.

I noticed that all of the mods were listed as being updated in the last day, so I'm wondering that it's just a weird build error or something similar.

I really like the way that you've made a number of simple modes that make specific contained changes rather than try and bundle everything together. Being able to go through the list and pick and choose the options I want is perfect.  I'm hoping that you'll update a number of your 1.16 mods to 1.17.1 soon so I can try those once this problem is sorted out.

Thanks for your time in reading this and your effort in creating the mods in the first place,
Ian Simcock.

**Comment by ricksouth** [comment]
Hi Ian,

Thanks for the kind words! Have you installed the latest Forge in your Minecraft instance?

**Comment by IanSimcock** [comment]
Hi Rick.

I hadn't installed Forge as none of the mods said that they needed it in the descriptions, nor did I get any error during the load.

I chased down the latest version of forge and added that to the vanilla install, but nothing changed. There are no messages reporting problems, but the two mods I could easily test, MoveBoats and PaperBooks, did not work. All the mods show in the mods section of the MineCraft Log, but there are no other references to them in the log.

Sorry I don't have any actual information to pass on. I know it's much harder to chase a problem when there isn't any crash or warning log to go on.

Ian Simcock.

UPDATE
---------
Hi again :-)

It turns out that Forge was having problems installing, and that was why they wouldn't work in a vanilla version. I finally sorted it out and got them to work.

Unfortunately I'm using Fabric on my current version of MineCraft, so I'll have to go without your mods.

Sorry for wasting your time,
Ian Simcock.
